crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -70,6 +70,21 @@ use crate::{
     rope::{Rope, RopeReadRef, RopeReader},
 };
 
+#[turbo_tasks::value(shared)]
+#[derive(Debug, Clone, Copy)]
+pub enum FileSystemKind {
```

Comment:
Some of the variants are implementation Details of other crates. It's also problematic to define more filesystems in other crates